package.json

```diff
@@ -34,13 +35,21 @@
     "@devtools-ds/object-inspector": "^1.1.2",
     "@devtools-ds/object-parser": "^1.2.1",
     "@devtools-ds/table": "^1.1.2",
-    "@devtools-ds/themes": "^1.1.2",
+    "@devtools-ds/themes": "^1.2.1",
+    "@devtools-ui/plugin": "0.0.2--canary.13.662",
     "@emotion/styled": "^11",
     "@oclif/core": "1.9.0",
     "@oclif/plugin-legacy": "^1.2.7",
     "@oclif/plugin-plugins": "^1.9.0",
-    "@player-ui/player": "0.7.1-next.0",
-    "@player-ui/types": "0.7.1-next.0",
+    "@player-tools/dsl": "^0.5.1",
```

Comment:
I don't think we need to pull this in from remote

Reply:
Yeap, removed.